index.md

```diff
@@ -15,7 +15,7 @@ plagued reflection-based solutions. More details can be found in
 
 ## Where are the docs?
 
-* [User documentation](users-guide)
+* [User documentation](users-guide.html)
```

Comment:
Ah, sorry, I actually did initial testing with an out-of-date `jekyll/jekyll:pages` Docker image where extensionless URLs didn't work; and I forgot to re-test after doing a `docker pull` (BTW, would you be interested in a `docker-compose.yml` to easily test locally?). Will remove extensions everywhere, as I originally did.